src/screencast/screencast_common.c

```diff
@@ -1,18 +1,28 @@
 #include "screencast_common.h"
 
-enum spa_video_format pipewire_from_wl_shm(void *data) {
-	struct screencast_context *ctx = data;
+void randname(char *buf) {
```

Comment:
Hmm, this is a little bit dangerous, since this relies on `buf` having at least 6 bytes. Maybe at least add `assert(buf[i] == 'X')` in the loop?